nil/services/nil_load_generator/jsonrpc.go

```diff
@@ -1,12 +1,50 @@
 package nil_load_generator
 
 import (
+	"context"
+	"crypto/rand"
+	"errors"
+	"math/big"
+
+	"github.com/NilFoundation/nil/nil/common"
 	"github.com/NilFoundation/nil/nil/internal/types"
+	"github.com/NilFoundation/nil/nil/services/cliservice"
+	uniswap "github.com/NilFoundation/nil/nil/services/nil_load_generator/contracts"
 )
 
 type NilLoadGeneratorAPI interface {
 	HealthCheck() bool
 	SmartAccountsAddr() []types.Address
+	CallSwap(tokenName1, tokenName2 string, amountSwap, expectedAmount uint64) (common.Hash, error)
+	CallQuote(tokenName1, tokenName2 string, swapAmount uint64) (*big.Int, error)
+	CallInfo(hash common.Hash) (UniswapTransactionInfo, error)
```

Comment:
It doesn't work now with in flight for progress. Transaction not found happens. It's only will work when all hapens.